addon/session-stores/cookie.js

```diff
@@ -148,6 +161,8 @@ export default BaseStore.extend({
   init() {
     this._super(...arguments);
 
+    this.set('_cookiePath', Configuration.baseURL);
```

Comment:
tests in `cookie-store-behaviour.js` expect that cookie path is set to '/' e.g.
https://github.com/simplabs/ember-simple-auth/blob/master/tests/unit/session-stores/shared/cookie-store-behavior.js#L110

I guess there are 2 ways to sort this out - either change it in all asserts from `path === '/'` to `path === Configuration.baseURL` or just set a default baseURL for all tests (which I did)